geode-core/src/test/java/org/apache/geode/internal/cache/entries/AbstractRegionEntryTest.java

```diff
@@ -14,10 +14,13 @@
  */
 package org.apache.geode.internal.cache.entries;
 
+import static org.apache.geode.internal.Assert.assertTrue;
```

Comment:
instead of modifying the AbstractRegionEntry unit test I think you should modify or add a unit test for DiskStoreID. That is the class you are modifying and it should make your test much simpler since you can just directly test compareTo and demonstrate that it returns 0 for DiskStoreID that are not equal.